test-data-helpers/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+db.side = client
```

Comment:
Try to move setup closer to one of the db-sides so the application properties is close to the module using the testdata.